portfolio/src/main/webapp/index.html

```diff
@@ -51,18 +51,19 @@ <h1>Contact Me:</h1>
           </tr>
       </table>
 
-      <form action="/form-handler" method="POST">
+      <form action="/form-handler" method="POST" style="margin-top: 10px; margin-bottom: 10px;">
```

Comment:
optional: instead of adding the `style=` on the html element, instead do `class=` and add those styles to a new css class. Css classes help keep the html a little cleaner (imagine if you had 10 style attributes on this guy, it'd look a lot worse) and also lets you reuse some styles

Reply:
I agree, I just figured I'd add the style here since it wasn't going to be a lot of styling.